src/data/gitmojis.json

```
@@ -503,6 +503,14 @@
       "description": "Data exploration/inspection.",
       "name": "monocle-face",
       "semver": null
+    },
+    {
+      "emoji": "⚰️",
+      "entity": "&#x26B0;",
+      "code": ":coffin:",
+      "description": "Remove dead code",
```

Just a little think to fix: All descriptions should end with a dot 😉  

```suggestion
      "description": "Remove dead code.",
```